crates/config/src/config.rs

```diff
@@ -294,6 +294,31 @@ impl Default for PruneConfig {
     }
 }
 
+impl PruneConfig {
+    /// Converts a [`PruneConfig`] into an usable [`PruneTargets`].
+    pub fn into_targets(&self, head: Option<u64>) -> PruneTargets {
```

Comment:
1. Does it need to be `Option`? I see we pass `Some` now all the time
2. I think we can use `BlockNumber` here instead of `u64`

Reply:
the one on pipeline itself is an Option<BlockNUmber> though (`max_block`)